{{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/{{cookiecutter.django_default_app_name}}/tests/test_database.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from django.db import models
+
+pytestmark = pytest.mark.django_db
+
+
+class DummyValue(models.Model):
```

Comment:
it seems this only works because there are no migrations in this project

If someone uses this template as is, as soon as they create migrations they will have to debug this broken tests which is not a good experience.

Reply:
Ack, now using User model instead